predictive_models/baseline.py

```diff
@@ -0,0 +1,51 @@
+"""A Baseline model
+Source: https://www.tensorflow.org/tutorials/structured_data/time_series
+"""
+
+import tensorflow as tf
+import pandas as pd
+from window_generator import WindowGenerator
+
+
+# pylint: disable=too-few-public-methods
+class Baseline(tf.keras.Model):
+    def __init__(self, label_index=None):
+        super().__init__()
+        self.label_index = label_index
+
+    def call(self, inputs):
+        if self.label_index is None:
+            return inputs
+        result = inputs[:, :, self.label_index]
+        return result[:, :, tf.newaxis]
+
+
+dir_path = "/Users/seppas/Documents/Moody+/SphereDecay/"
+CFD_5 = pd.read_csv(dir_path + "05SphereDecayCFD_V3", header=0)
+CFD_3 = pd.read_csv(dir_path + "03SphereDecayCFD_V3", header=0)
+CFD_1 = pd.read_csv(dir_path + "01SphereDecayCFD_V3", header=0)
+train_df = CFD_5
+val_df = CFD_3
+test_df = CFD_1
+column_indices = {name: i for i, name in enumerate(train_df.columns)}
+num_features = train_df.shape[1]
+
+w2 = WindowGenerator(
+    input_width=32,
+    label_width=1,
+    shift=1,
+    train_df=train_df,
+    val_df=val_df,
+    test_df=test_df,
+)
+
+baseline = Baseline()
+
+baseline.compile(
+    loss=tf.losses.MeanSquaredError(), metrics=[tf.metrics.MeanAbsoluteError()]
+)
+
+val_performance = {}
+performance = {}
+val_performance["Baseline"] = baseline.evaluate(w2.val)
+performance["Baseline"] = baseline.evaluate(w2.test, verbose=0)
```

Comment:
We don't need to declare the `dict` first, we can simplify to:
```python
val_performance = {"Baseline": baseline.evaluate(w2.val)}
performance = {"Baseline": baseline.evaluate(w2.test, verbose=0)}
```
However, the `dict`s don't seem to have any other entry. So maybe we should do:
```python
val_performance = baseline.evaluate(w2.val)
performance = baseline.evaluate(w2.test, verbose=0)
```
Also, `val_performance` is not clear. Is is `validation_performance`? If so, let's spell it out and avoid abbreviations.